libs/sdk-ui-kit/src/Header/HeaderInviteButton.tsx

```diff
@@ -0,0 +1,16 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { useIntl } from "react-intl";
+import { IHeaderInviteButtonProps } from "./typings";
+
+export const HeaderInviteButton: React.FC<IHeaderInviteButtonProps> = ({ onInviteButtonClick }) => {
+    const intl = useIntl();
+    return (
+        <div className="gd-invite-button" onClick={onInviteButtonClick}>
```

Comment:
If the item is not a button, should we remove `button` in `gd-invite-button`? Something like:

`gd-invite-button` → `gd-header-invite`